src/style.css

```diff
@@ -1 +1,125 @@
 /* add styles here */
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 600;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 8px;
+  padding: 0;
+  font-size: 16px;
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+}
+
+.search--form {
+  display: flex;
+  justify-content: center;
+  margin: 20px auto 0;
+}
```

Comment:
Be consistent with your margins. It's recommended to add only top or bottom margins but not both to avoid potential margin collapse. In this case, you've set a top margin on '.search--form'. Ensure that this does not cause any issues with adjacent elements.